src/App.js

```diff
@@ -5,17 +5,18 @@ import styled from 'styled-components/macro'
 import TimeRemaining from './components/TimeRemaining'
 import Modal from 'react-modal'
 import logo from './components/img/logo.png'
-import { cardsRef } from './firebase'
```

Comment:
just out of curiosity: Are you still planning to integrate firestore as a database later?

Reply:
Oh no, that's a mistake. I'll delete this line, thanks for asking